client/src/components/Login.js

```diff
@@ -57,8 +57,24 @@ signupLink:{
 }));
 function Login (){
     const classes = useStyles();
-    const [userEmail,setEmail]=useState('')
-    const [userPass, setPass]=useState('')
+    const [email,setEmail]=useState('')
+    const [password, setPass]=useState('')
+    const handleClick = () => {
```

Comment:
A couple small cleanup notes:
- let's actually makes this setPassword to be consistent
- let's add semi colons 
- add a line break between the state and first function, for readability